packages/lib/gpcircuits/src/proto-pod-gpc.ts

```diff
@@ -143,6 +163,21 @@ export type ProPODGPCCircuitDesc = CircuitDesc & {
    */
   maxEntries: number;
 
+  /**
```

Comment:
nit: Why insert these between maxEntries and merkleMaxDepth rather than at the end?
This is somewhat arbitrary, but I prefer to keep these grouped conceptually, and the merkle depth applies to the entries, so I'd rather keep it directly after maxEntries (here and elsewhere like in the circuit names).
We might actually need to rename merkeMaxDepth to podMerkleMaxDepth if/when we have a group membership check with its own merkle proofs.